packages/react-core/src/components/Nav/examples/Nav.md

```diff
@@ -68,6 +71,12 @@ A flyout should be a `Menu` component. Press `space` or `right arrow` to open a
 
 ```
 
+### Link text
```

Comment:
Nit: I feel like the example name could be tweaked a little, something like "With text and icon" or "Custom link text" or something. Not sure if "link text" fully conveys what the example is showing

Reply:
I don't know that the use case is specifically for an icon.  You can pass any react node.